addons/web/static/src/core/errors/error_dialogs.xml

```diff
@@ -68,4 +68,15 @@
       </Dialog>
     </t>
 
+    <t t-name="web.ScssErrorDialog" owl="1">
+      <Dialog title="this.constructor.title">
+        <div role="alert">
+          <p>
+            The style compilation failed, see the error below. Your recent actions may be the cause, please try reverting the changes you made.
```

Comment:
This message is not correct: think about a user on his day to day work seeing this message because the admin fiddled with the styles during the day in production (think odoo.sh), he will think his last invoice caused the error and will try to fix it after the admin has done the actual fixing.
I would replace it by something along the lines of 
```The style compilation failed, see the error below. This is an administrator or developer error that must be fixed for the entire database before continuing working.``

Reply:
I agree that it's poor wording, I've kept the message as it was but since I'm modifying this anyway I might as well improve it.